### PR TITLE
fix: Login screen updates

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [MOPLAT-373]

image

https://user-images.githubusercontent.com/17421923/187421869-1d1e63bd-650b-416b-8f03-4a49a0d22404.mov

https://user-images.githubusercontent.com/17421923/187422036-316b11a6-d41b-4333-b7dc-4ad1ef9b6071.mov

### PR Checklist

- [ ] I tested my changes on **iOS** and **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

Need help with something? Have a look at our [docs], or get in touch with us.

[docs]: /docs/README.md

[MOPLAT-373]: https://artsyproduct.atlassian.net/browse/MOPLAT-373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/screens/Auth/Login.tsx [hunk 1]
```diff
@@ -92,143 +99,158 @@ export const LoginScreen: React.FC = () => {
...
+        >
+          <Flex flex={1}>
+            <Text variant="lg">Folio</Text>
+            <Spacer mt={2} />
+            <Text variant="xl">Log In</Text>
+            <Text variant="md" mt={0.5}>
+              With Your Artsy Partner Account
+            </Text>
+            <Spacer mt={4} />
+            <Box>
```
Comment (patrinoua) [inline src/app/screens/Auth/Login.tsx:126]: Can we use Flex instead of Box?

Comment (patrinoua) [reply src/app/screens/Auth/Login.tsx:126]: Suggesting this as we are only using Box in one other component (TabBarComponent) and I think in general we wanted to shift towards using Flex in general

Comment (rajsam003) [reply src/app/screens/Auth/Login.tsx:126]: Good point

**Review by brainbicycle — APPROVED** [review]
thanks for the follow-ups!

**Pull request merged**